salt/_macros/certs.jinja

```diff
@@ -29,13 +29,13 @@
                        "kubernetes.default.svc",
                        "api",
                        "api." + pillar['internal_infra_domain'],
-                       salt['pillar.get']('api:server:external_fqdn', ''),
-                       salt['pillar.get']('api:cluster_ip', '')] +
-                       salt['pillar.get']('api:server:extra_names', []) +
-                       salt['pillar.get']('api:server:extra_ips', []) +
+                       salt.caasp_pillar.get('api:server:external_fqdn'),
+                       salt.caasp_pillar.get('api:cluster_ip')] +
```

Comment:
Typo? `]`

Reply:
No, it is concatenating lists. For example, with default values, it will be `[... + salt.caasp_pillar.get('api:cluster_ip')] + [] + []`

Reply:
Okay, the diff in github has a partial context. Thanks!